config.schema.yml

```diff
@@ -304,7 +304,7 @@ properties:
                                     type: "string"
                                 aliasTemplate:
                                     type: "string"
-                                    pattern: "^#.*\\$CHANNEL"
+                                    pattern: "^(#.*)?\\$CHANNEL"
```

Comment:
* The pattern does not include the mandatory hash sign from a Matrix room ID?
* Why do we need the optional group? Can $CHANNEL not start with a hash sign?

Reply:
Hmm, I guess it may come up. Will rethink this one

Reply:
Perhaps the solution is to have `($HASH_CHANNEL|#$CHANNEL)`.

Reply:
Practically speaking this appears to work. 

>The pattern does not include the mandatory hash sign from a Matrix room ID?

The $CHANNEL starts with a hash, so in the case of #matrix:libera.chat, the hash prefix would make this ##matrix:libera.chat.

>Why do we need the optional group? Can $CHANNEL not start with a hash sign?

Because it's valid to have #matrix, ##_irc_#matrix and ##matrix as your localpart, but never foo_#matrix.